data.tf

```diff
@@ -15,14 +15,34 @@ data "aws_iam_policy_document" "workers_assume_role_policy" {
   }
 }
 
-data "aws_ami" "eks_worker" {
+data "aws_ami" "eks_default" {
   filter {
     name   = "name"
     values = ["amazon-eks-node-*"]
   }
 
   most_recent = true
-  owners      = ["602401143452"] # Amazon
+  owners      = ["602401143452"]
+}
+
+data "aws_ami" "eks_gpu" {
```

Comment:
Again, not so much a fan of splatting Amazon as many times as we can across a line. I think anyone looking at this will find it (painfully) obvious we're working in Amazon Web Services.